GW Launcher/Forms/MainForm.cs

```diff
@@ -203,39 +203,39 @@ private void notifyIcon_MouseClick(object sender, MouseEventArgs e)
                 rightClickOpen = false;
                 return;
             }
+
             rightClickOpen = true;
         }
 
         var isVisible = (Point p) =>
         {
-            return Screen.AllScreens.Any(s => p.X < s.Bounds.Right && p.X > s.Bounds.Left && p.Y > s.Bounds.Top && p.Y < s.Bounds.Bottom);
+            return Screen.AllScreens.Any(screen => p.X < screen.Bounds.Right && p.X > screen.Bounds.Left && p.Y > screen.Bounds.Top && p.Y < screen.Bounds.Bottom);
```

Comment:
do not rename variables in one-liner lambda functions please, this just makes it harder to read in my opinion